examples/00-EDB/01_edb_example.py

```diff
@@ -131,41 +167,42 @@
 setup.set_dc_slider = 0
 setup.add_source_terminal_to_ground("V1", 1)
 
-
-
 ###############################################################################
-# Save modifications
-# ~~~~~~~~~~~~~~~~~~
-# Save modifications.
+# ## Solve
+#
+# Save the modifications and run the analysis in SIwave.
 
 edb.save_edb()
 edb.nets.plot(None, "1_Top",plot_components_on_top=True)
 
 siw_file = edb.solve_siwave()
 
 ###############################################################################
-# Export Siwave Reports
-# ~~~~~~~~~~~~~~~~~~~~~
-# Export all DC Reports quantities.
+# ## Export Results
+#
+# Export all quantities calculated from the DC-IR analysis. The following method runs SIwave in batch mode from the command line. Results are written to the edb folder.
 outputs = edb.export_siwave_dc_results(siw_file, setup.name, )
 
 ###############################################################################
-# Close EDB
-# ~~~~~~~~~
-# Close EDB. After EDB is closed, it can be opened by AEDT.
+# Close the EDB. After EDB is closed, it can be opened by AEDT.
 
 edb.close_edb()
 
 ###############################################################################
-# Postprocess in Siwave
-# ~~~~~~~~~~~~~~~~~~~~~
-# Open Siwave and generate a report. This works on Window only.
+# ## View Layout in SIwave
+#
+# The SIwave user interface can be visualized and manipulated
+# using the SIwave user interface. This command works on Window OS only.
```

Comment:
```suggestion
# ## View layout in SIwave
#
# You can visualize and manipulate the SIwave user interface
# using the SIwave user interface. This command works on the Window OS only.
```
Not sure why "using the SIwave user interface is necessary for visualizing and manipulating the SIwave user interface?